docker-compose-setup/docker-compose.yml

```
@@ -0,0 +1,54 @@
+services:
+  coatrack-postgres:
+    image: postgres:9.4.26
+    container_name: coatrack-postgres
+    volumes:
+      - coatrack-postgres:/var/lib/postgresql/data
+    environment:
+      POSTGRES_PASSWORD: password1234
```

I am wondering about a better password. Maybe use something like:
```suggestion
      POSTGRES_PASSWORD: insert-postgres-password-here
```
or
```suggestion
      POSTGRES_PASSWORD: postgres-password-placeholder
```
??

---

Done. I used the first suggestion.